src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -80,6 +84,9 @@ const LibrariesTodayHours = ({libraries, ...props}: { libraries: HoursProps["lib
               {library?.suLibraryHours &&
                 <TodayLibraryHours branchId={library.suLibraryHours}/>
               }
+              {library?.map &&
+                <span className="float-right w-auto inline"><Link href={library?.map} ><span className="sr-only">{library.title} Location</span><MapPinIcon title="Map" width={25} className="mr-5 inline"/>Location</Link></span>
```

Comment:
Now you have 2 "location" strings. either wrap the second in `aria-hidden` or remove the first one. Ensure there is a space though. Sometimes you have to use `&nbps;` to force a space at the end of tag.

Reply:
I removed the first one since the second is visible, and included the &nbps; for safety.  Also got that git hook for `prettier` to do its thing correctly.